openapi/openapi.yaml

```diff
@@ -999,6 +976,176 @@ components:
           required:
             - data
 
+    UiCollection:
+      title: 'UiCollection'
+      description: 'Detailed info about a collection used by the ui'
+      type: object
+      properties:
+        all_versions:
+          readOnly: true
+          type: array
+          items:
+            $ref: '#/components/schemas/UiCollectionVersion'
+        community_score:
```

Comment:
We're not adding surveys to automation hub so we can drop community score and survey count